invenio_migrator/legacy/records.py

```diff
@@ -95,12 +110,17 @@ def get(query, from_date, **kwargs):
     return len(recids), recids
 
 
-def dump(recid, from_date, with_json=True, latest_only=False, **kwargs):
+def dump(recid, from_date, with_json=True, latest_only=False,
```

Comment:
I think you were looking at an old version, the default value is `with_json=False`